src/main/java/com/extendedclip/papi/expansion/player/PlayerUtil.java

```diff
@@ -136,4 +136,31 @@ public static int getEmptySlots(Player p) {
         }
         return slots;
     }
+    
+    public static int getExperienceAtLevel(Player player) {
+        return getExperienceAtLevel(player.getLevel());
+    }
```

Comment:
Any reason for making a new method for this and it's used once? and it's public while it's only used in this class